sys-fs/cryptmount/cryptmount-5.3.1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-info systemd
+
+DESCRIPTION="A utility for management and user-mode mounting of encrypted filesystems"
+HOMEPAGE="http://cryptmount.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="argv0switch cswap fsck gcrypt largefile mount nls luks openssl systemd udev"
+
+RDEPEND="
+	gcrypt? ( dev-libs/libgcrypt:0= )
+	nls? ( sys-devel/gettext )
+	luks? ( sys-fs/cryptsetup )
+	openssl? ( dev-libs/openssl:0= )
+	systemd? ( sys-apps/systemd )
+	virtual/libiconv
+	virtual/libintl
+"
+
+DEPEND="
+	${RDEPEND}
+	sys-kernel/linux-headers
+"
+
+DOCS=(
```

Comment:
no need to expand this over multiple lines.

Reply:
Ok.